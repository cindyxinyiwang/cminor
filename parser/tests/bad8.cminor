// invalid function

hello1: function integer () = {};
