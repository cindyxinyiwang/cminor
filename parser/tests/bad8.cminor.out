parse error: ';' not allowed after function declaration
